### DISCUSSION TITLE
Details list multi selection filter (with text only values) and formatting of date without AddColumn function

### DISCUSSION THREAD

**Discussion by vito924** [microsoft/powercat-creator-kit#321] [Q&A] [answered]
Hello everyone
I am having trouble with Creators Kit Details list; 
I want to make multi selection filter (with text only values) and format date (to Europe format) without AddColumn function.

Please write, any suggestions are welcome to find workaround. 

Thanks in advance!

<img width="769" alt="Snimka zaslona 2023-04-25 151540" src="https://user-images.githubusercontent.com/93899848/234288240-0e4aacc3-2fc1-4389-8c31-5a19dc95154a.png">

**Comment by yaseralhosani** [comment, marked as answer]
You can check video below 

https://youtu.be/l8v4W-A9-ao

**Reply by vito924** [reply]
Thanks for the reply, I will try with it, but I have text only fields. We'll see.
